hubspot3/globals.py

```diff
@@ -1,7 +1,7 @@
 """
 globals file for hubspot3
 """
-__version__ = "3.2.36"
+__version__ = "3.2.37"
```

Comment:
Normally I'll handle updating this version number, as I need to manually trigger a new release to pip